mdata/store_cassandra.go

```diff
@@ -301,25 +396,25 @@ func (c *cassandraStore) Search(key string, start, end uint32) ([]chunk.IterGen,
 
 	row_key := fmt.Sprintf("%s_%d", key, start_month/Month_sec)
 
-	query(start_month, start_month, "SELECT ts, data FROM metric WHERE key=? AND ts <= ? Limit 1", row_key, start)
+	query(start_month, start_month, fmt.Sprintf("SELECT ts, data FROM %s WHERE key=? AND ts <= ? Limit 1", table), row_key, start)
```

Comment:
are you sure we need fmt.Sprintf? seems like we can just pass the table name as another argument. when the query is executed, it will evaluate all arguments and safely insert them into the statement string.